fsm/fsm.go

```diff
@@ -10,6 +10,7 @@ import (
 // NewFSM returns new FSM implementation based on initial state
 func NewFSM(initialState int64, log *zap.Logger) *Fsm {
 	return &Fsm{
+		// maxJobs per process is here?
```

Comment:
No, this is a bad place for that. Since this is a Finite-state machine for the worker.